spec/gilded_rose_spec.rb

```diff
@@ -74,13 +83,12 @@
       expect(items[3].quality).to eq (0)
     end
   
-    # NEED: to deal with conjured items
-    # it "conjured items degrade in Quality twice as fast as normal items" do
-    #   items = [Item.new("conjured", 50, 50), Item.new("conjured", 0, 50)]
-    #   GildedRose.new(items).update_quality()
-    #   expect(items[0].quality).to eq (48)
-    #   expect(items[1].quality).to eq (46)
-    # end
+    it "conjured items degrade in Quality twice as fast as normal items" do
```

Comment:
This is a great test 👍🏽 What are your thoughts on creating separate variables for each conjured item for readability? For example `conjured_item` and `conjured_item_expired` or something along those lines

Reply:
That's a great idea! I will add that to my incoming PR. Thanks @ellehallal!